web-admin/src/routes/[organization]/[project]/-/share/[token]/+page.ts

```diff
@@ -1,17 +1,43 @@
-import { getExploreStates } from "@rilldata/web-common/features/explores/selectors";
+import { fetchMagicAuthToken } from "@rilldata/web-admin/features/projects/selectors";
+import { ResourceKind } from "@rilldata/web-common/features/entity-management/resource-selectors.js";
+import { error, redirect } from "@sveltejs/kit";
 
-export const load = async ({ url, parent }) => {
-  const { explore, metricsView, defaultExplorePreset, token } = await parent();
-  const exploreName = token?.resourceName;
-  const metricsViewSpec = metricsView.metricsView?.state?.validSpec;
-  const exploreSpec = explore.explore?.state?.validSpec;
+export const load = async ({
+  params: { organization, project, token },
+  url,
+}) => {
+  // Public URLs specify the resource in the token's metadata
+  const tokenResp = await fetchMagicAuthToken(token).catch((e) => {
+    console.error(e);
+    throw error(404, "Unable to find token");
+  });
 
-  return getExploreStates(
-    exploreName,
-    `${token.id}__`,
-    url.searchParams,
-    metricsViewSpec,
-    exploreSpec,
-    defaultExplorePreset,
+  const { token: tokenMetadata } = tokenResp;
+  if (!tokenMetadata?.resources) {
```

Comment:
Are we migrating all tokens for existing public urls? If not then we need to check `resourceName` for backwards compatibility.

Reply:
Yes: 
<img width="719" alt="image" src="https://github.com/user-attachments/assets/e634453d-e23e-4ff2-9b0e-6649b48240b4" />